template/src/components/Row.tsx

```diff
@@ -1,4 +1,4 @@
-import React, {FC, ReactElement, ReactNode} from 'react'
+import React, {ReactElement} from 'react'
```

Comment:
unused

Reply:
Can you explain in detail. I don't get you point

Reply:
if use FunctionComponent :FC<Props>
Always exist `children` prop

Reply:
we are using version react: 18, so it don't include the children props it's self. 
https://solverfox.dev/writing/no-implicit-children/

Reply:
thanks for updating